deploy/api/dynamic-import.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Dynamic import"
+---
+
+Deno Deploy supports [dynamic import] but with some limitations. This page
```

Comment:
is this supposed to be a link? Or in backticks?

Reply:
It's rendered as a link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import
![render result](https://github.com/user-attachments/assets/cd4b475a-9719-4be9-8e60-c209b9f57c35)

Reply:
@thisisjofrank look at the bottom of the file. This syntax can be used to refer to links there so that you don't have to type them over and over again when you want to have the same link multiple times on your page.